dialogue-clients/src/test/java/com/palantir/dialogue/otherpackage/DialogueClientsTest.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.dialogue.otherpackage;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.palantir.conjure.java.api.config.service.PartialServiceConfiguration;
+import com.palantir.conjure.java.api.config.service.ServiceConfiguration;
+import com.palantir.conjure.java.api.config.service.ServicesConfigBlock;
+import com.palantir.conjure.java.api.config.service.UserAgent;
+import com.palantir.conjure.java.client.config.ClientConfiguration;
+import com.palantir.conjure.java.client.config.NodeSelectionStrategy;
+import com.palantir.conjure.java.clients.ConjureClients;
+import com.palantir.dialogue.TestConfigurations;
+import com.palantir.dialogue.clients.DialogueClients;
+import com.palantir.dialogue.example.SampleServiceAsync;
+import com.palantir.dialogue.example.SampleServiceBlocking;
+import com.palantir.logsafe.Preconditions;
+import com.palantir.refreshable.Refreshable;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
+import java.security.Provider;
+import org.junit.jupiter.api.Test;
+
+class DialogueClientsTest {
+
+    ServiceConfiguration serviceConf = ServiceConfiguration.builder()
```

Comment:
nit
```suggestion
    private static final ServiceConfiguration serviceConf = ServiceConfiguration.builder()
```

Reply:
This gets assigned from the `@BeforeEach` method so can't be static/final sadly.

Reply:
I don’t think this test has a before-each, isn’t this assigned when an instance of the test class is created?

Reply:
Ah whoops commented on the wrong one - this one is fixed now.